src/data_loading/chernobyl.py

```diff
@@ -0,0 +1,31 @@
+"""
```

Comment:
Shall we rename this script to `utils.py` or `loading_utils.py` as we might want to add more loading routines?

Reply:
Haha good catch, that module used to have more than one function but now with just one it doesn't make really sense anymore. My suggestion would actually be to make it a `jasmin.py` module that includes data loaders for data on our group workspace. When loading the module we can include a check if the user has access to the workspace and throw a warning if not.

Reply:
Done, renamed it to `jasmin.py`. See f723bd6